code/modules/mob/living/living_defines.dm

```diff
@@ -74,6 +74,7 @@
 	var/fire_stacks = 0 //Tracks how many stacks of fire we have on, max is
 
 	var/chestburst = 0 // 0: normal, 1: bursting, 2: bursted.
+	var/headbite = 0 //0: normal 1: brain removed
```

Comment:
If its a binary between 1 and 0 use TRUE and FALSE, also might consider mass replacing it to something like headbitten so its more clear